ui/src/message_popup/playtest/hmtca_experience_page.jsx

```diff
@@ -143,18 +159,21 @@ export default React.createClass({
   },
 
   renderContent() {
-    const {questions, didSkipAhead} = this.state;
-    if (!questions) return this.renderIntro();
-    if (didSkipAhead) return this.renderGroupReview();
-
-    return <LinearSession
-      questions={questions}
-      questionEl={this.renderQuestionEl}
-      summaryEl={this.renderClosingEl}
-      onLogMessage={this.onLogMessage}
-    />;
-  },
+    const {questions, currentPart} = this.state;
+    if (currentPart === Parts.PRACTICE) {
+      if (!questions) return this.renderIntro();
+      return <LinearSession
+        questions={questions}
+        questionEl={this.renderQuestionEl}
+        summaryEl={this.renderPauseEl}
+        onLogMessage={this.onLogMessage}
+      />;
+    }
 
+    if (currentPart === Parts.GROUP_INSTRUCTIONS) return this.renderGroupInstructions();
+    if (currentPart === Parts.GROUP_REVIEW) return this.renderGroupReview();
+    if (currentPart === Parts.FINAL_INSTRUCTIONS) return this.renderFinalInstructions();
```

Comment:
@milsim11 I revised this from what we were doing with the `isAllDone` etc. variables earlier, so that `currentPart` tracks what phase we should be in, and updates it accordingly.

Moving the pause screen into here makes that simpler, because for the `LinearSession` component, the pause screen is now the "summary" shown at the end of all the questions.